features/multivariate_put_dimension.authorized.feature

```diff
@@ -0,0 +1,185 @@
+        },
+        {
+          "name": "geography",
+          "id": "country",
+          "label": "Country",
+          "options": [],
+          "is_area_type": true
+        }
+        ],
+        "dataset": {
+          "id": "cantabular-example-1",
+          "edition": "2021",
+          "version": 1
+        },
+        "published": true,
+        "population_type": "dummy_data_households",
+        "type": "multivariate"
+      }
+    ]
+    """
+
+    And Cantabular returns these dimensions for the dataset "dummy_data_households" and search term "hh_carers":
+    """
+    {
+      "dataset": {
+        "variables": {
+          "edges": [
+            {
+              "node": {
+                "categories": {
+                  "totalCount": 32
+                },
+                "label": "Number of unpaid carers in household (32 categories)",
+                "mapFrom": [],
+                "name": "hh_carers"
+              }
+            }
+          ]
+        }
+      }
+    }
+    """
+
+   Scenario: Replacing a non-goegraphy filter dimension
+    When I PUT "/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions/siblings"
+    """
+    {
+      "name": "hh_carers",
+      "id": "hh_carers",
+      "is_area_type": false
+    }
+    """
+    And I GET "/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions"
+
+    Then I should receive the following JSON response:
+    """
+    {
+      "items": [
+        {
+          "name": "geography",
+          "id": "country",
+          "label": "Country",
+          "links": {
+            "filter": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1",
+              "id": "94310d8d-72d6-492a-bc30-27584627edb1"
+            },
+            "options": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions/geography/options"
+            },
+            "self": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions/geography",
+              "id": "country"
+            }
+          }
+        },
+        {
+          "name": "hh_carers",
+          "id": "hh_carers",
+          "label": "Number of unpaid carers in household (32 categories)",
+          "links": {
+            "filter": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1",
+              "id": "94310d8d-72d6-492a-bc30-27584627edb1"
+            },
+            "options": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions/hh_carers/options"
+            },
+            "self": {
+              "href": "http://localhost:22100/filters/94310d8d-72d6-492a-bc30-27584627edb1/dimensions/hh_carers",
+              "id": "hh_carers"
+            }
+          }
+        }
+      ],
+      "limit": 20,
+      "offset": 0,
+      "count": 2,
+      "total_count": 2
+    }
+    """
```

Comment:
It looks like a new line is needed at the end of this file.

Reply:
New line now added